### PR TITLE
Make_event_json factory function unnecessary.

### PR DESCRIPTION
**Pull request by eccles**
Problem:
The make_event_json factory function serves no puprose except to make
reuse difficult.

Solution:
Replace with MyAsset class with methods that reflect user's expected
use cases such as 'update_firmware' etc....
Additionally make all events use Recordevidence' behaviour.

Signed-off-by: Paul Hewlett <phewlett76@gmail.com>

### CODE REVIEW

File: archivist_samples/testing/asset.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+from .namespace import (
+    events_create,
+)
+
+CONFIG_MANAGEMENT = "Config Management"
+FW_UPDATE = "FW Update"
+FW_VULNERABILITY = "FW Vulnerability"
+OPERATIONAL_REPORT = "Operational Report"
+SERVICE_RP = "Service RP"
+SERVICE_RQ = "Service RQ"
```
Comment (JAG-UK) [inline archivist_samples/testing/asset.py:35]: given how they're used above these should match the pairs we agreed for the SaaS service updates: 
* 'Vulnerability Report'
* 'Vulnerability Addressed'
* 'Maintenance Request'
* 'Maintenance Performed'

File: archivist_samples/door_entry/run.py [hunk 2]
```diff
@@ -48,9 +48,6 @@
 
 BEHAVIOURS = [
     "Attachments",
```
Comment (JAG-UK) [inline archivist_samples/door_entry/run.py:50]: Is the Attachments behaviour still necessary? Can't you achieve that with `event.asset_attributes.attachments`?

**Review by JAG-UK — CHANGES_REQUESTED** [review]

File: archivist_samples/door_entry/run.py (hunk 2 above)
Comment (eccles) [reply archivist_samples/door_entry/run.py:50]: well the doors do have attachments and the attachments behaviour is still valid?

File: archivist_samples/testing/asset.py (hunk 1 above)
Comment (eccles) [reply archivist_samples/testing/asset.py]: fixed

**Review by JAG-UK — APPROVED** [review]

**Pull request merged**